### PR TITLE
Fix issue with pack in fixtures (installer)

### PR DESCRIPTION
**Pull request by alegout**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.3.x
| Description?  | Fix issue product is not pack (exception on front)
| Type?         | bug fix
| Category?     | IN
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | [BOOM-5024](http://forge.prestashop.com/browse/BOOM-5024)
| How to test?  | Install a PrestaShop with demo products. Open the product which is a pack, It should work.

#### Important guidelines

* Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
* Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
* Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/8913)

### CODE REVIEW

File: src/PrestaShopBundle/Install/XmlLoader.php [hunk 1]
```diff
@@ -560,6 +560,20 @@ public function createEntityConfiguration($identifier, array $data, array $data_
         $this->storeId($entity, $identifier, $entity_id);
     }
 
+    /**
+     * @inheritdoc
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Install/XmlLoader.php]: {@inheritdoc} but if you sue this annotation you don't have to use @param and @return annotations: are you sure this is required here?

Comment (alegout) [reply src/PrestaShopBundle/Install/XmlLoader.php]: thanks!

**Review by mickaelandrieu — DISMISSED** [review]

**Comment by marionf** [comment]
@alegout 
I tried your PR but the product is still standard

**Review by mickaelandrieu — APPROVED** [review]

**Comment by eternoendless** [comment]
Thank you @alegout

**Pull request merged**